frontend/packages/process-editor/src/components/ConfigPanel/ConfigEndEvent/CustomReceiptContent/CreateCustomReceiptForm/SelectCustomReceiptDataModelId/SelectCustomReceiptDataModelId.tsx

```diff
@@ -17,37 +15,33 @@ export const SelectCustomReceiptDataModelId = ({
   onChange,
 }: SelectCustomReceiptDataModelIdProps): React.ReactElement => {
   const { t } = useTranslation();
-  const { layoutSets, existingCustomReceiptLayoutSetId, availableDataModelIds } =
-    useBpmnApiContext();
+  const { layoutSets, existingCustomReceiptLayoutSetId, allDataModelIds } = useBpmnApiContext();
 
   const existingDataModelId: string = getDataTypeFromLayoutSetsWithExistingId(
     layoutSets,
     existingCustomReceiptLayoutSetId,
   );
-
-  const dataModelOptions = getDataModelOptions(availableDataModelIds, existingDataModelId);
-
-  const availableDataModelIdsEmpty: boolean = dataModelOptions.length === 0;
+  const allDataModelIdsEmpty: boolean = allDataModelIds.length === 0;
 
   return (
     <StudioNativeSelect
```

Comment:
And now as the datatypeselect one are presented with from the `EditDataType` component now is a combobox, we might should change this to being a combobox as well?

Reply:
Agree 😄